backend/routers/apps.py

```diff
@@ -166,9 +166,11 @@ def get_plugin_capabilities():
             {'title': 'Memory Creation', 'id': 'memory_creation'},
             {'title': 'Transcript Processed', 'id': 'transcript_processed'},
         ]},
-        {'title': 'Proactive Notification', 'id': 'proactive_notification', 'scopes': [
+        {'title': 'Notification', 'id': 'proactive_notification', 'scopes': [
             {'title': 'User Name', 'id': 'user_name'},
-            {'title': 'User Facts', 'id': 'user_facts'}
+            {'title': 'User Facts', 'id': 'user_facts'},
+            {'title': 'Notification', 'id': 'notification'}
```

Comment:
why we need adding the new one `{'title': 'Notification', 'id': 'notification'}`

Reply:
my mistake, pls help me review again